app/experimenter/experiments/models/nimbus.py

```diff
@@ -300,6 +300,18 @@ def to_timestamp(date):
             slug=self.slug, from_date=start_date, to_date=end_date
         )
 
+    @property
+    def review_url(self):
+        is_staging = "settings-writer.stage" in settings.KINTO_ADMIN_URL
```

Comment:
This should definitely be in settings.py as a global constant

Reply:
as in you want a global `IS_STAGING` constant?